### ISSUE TITLE
How to deal with mismatching XLF versions in tea, documentation and TYPO3 core

### ISSUE THREAD

**Issue by moe2k** [TYPO3-Documentation/tea#198]
I've created this PR https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/1301 where I wanted to update the xliff version number in the example code in the documentation to 1.2, in order to reflect the best practice which is stated in this example extension here, since both belong to the documentation team: https://github.com/TYPO3-Documentation/tea/blob/main/Resources/Private/Language/locallang.xlf.

My PR got declined because TYPO3 Core doesn't use its own best practices, so the reasoning was not to update the example code. I understand that a mismatch between best practice and what the core does is not desirable. Ideally, those two whings should be in line. Still I would like to know what the rule is for documentation - do we only refer to best practices if they are obeyed in TYPO3 core? Or do we allow the documentation to differ between what's currently happening in TYPO3 and how things _should_ be by pointing to this extension here?

Either way, discussing that in the documentation channel, the question was raised whether XLF 1.2 is actually is considered best practice or not, and it was suggested to change the best practice here back to 1.0 or 1.1. 

Ideally, if we don't want to change the recommendation here, the TYPO3 language files should be updated to 1.2 and documentation also. In that case I'd create a ticket for TYPO3 core.

**Comment by oliverklee** [comment]
Yes, please create a ticket for the core. In case the ticket gets declined and the official recommendation is to use 1.0, then we can still downgrade the Tea language files.

**Comment by moe2k** [comment]
Thanks a lot!
FYI, I created the issue here: https://forge.typo3.org/issues/93594

**Comment by bmack** [comment]
Just some nice feedback here: Thank you everybody involved for making this happen in TYPO3 Core. You all rock!
